### ISSUE TITLE
transform-react-inline-elements doesn't work well with react-html-attrs

### ISSUE THREAD

**Issue by mayani** [babel/babel#5219]
babel-plugin-transform-react-inline-elements doesn't work well with babel-plugin-react-html-attrs

The react-html-attrs plugin transforms certain attributes like class to className. This makes it easier to copy HTML code directly as JSX. However, react-inline-elements plugin uses the untransformed value (class), instead of the transformed value (className). As a result. no class attribute gets set in the HTML generated by React.

### Input Code
```js
import React from "react";
export default class App extends React.PureComponent {
    render() {
        return (
            <div class="aaaa">
                Classname
            </div>
        );
    }
}
```

### Babel Configuration (.babelrc, package.json, cli command)
```js
{
  "presets": [
    "es2015",
    "react"
  ],
  "plugins": [
    "react-html-attrs",
    "transform-react-inline-elements"
  ]
}
```

### Expected Behavior

```js
_createClass(App, [{
        key: "render",
        value: function render() {
            return _jsx("div", {
                className: "aaaa"
            }, void 0, "Classname");
        }
    }]);
```

### Current Behavior

```js
 _createClass(App, [{
        key: "render",
        value: function render() {
            return _jsx("div", {
                "class": "aaaa"
            }, void 0, "Classname");
        }
    }]);
```

### Your Environment

| software         | version
| ---------------- | -------
| Babel            |  6.22.2
| node             | 6.9.1
| npm              | 4.1.1
| Operating System | MacOSX 10.11.6

**Comment by hzoo** [comment]
Don't think babel-plugin-react-html-attrs is officially supported in anyway, but we'd accept a PR if it doesn't break anything

**Comment by mayani** [comment]
Fixed in insin/babel-plugin-react-html-attrs#9
